apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -1730,7 +1729,88 @@ async fn router(
             workflows_terminate_route(req, is_prod, project.clone(), name.to_string()).await
         }
         (_, &hyper::Method::OPTIONS, _) => options_route(),
-        _ => route_not_found_response(),
```

Comment:
this catchall handler enables matching all routes a user may specify in their byof app.

it falls through to the `route_not_found_response()` if none match